darts/timeseries.py

```diff
@@ -669,6 +669,39 @@ def shift(self, n: int) -> 'TimeSeries':
         new_series.index = new_time_index
         return TimeSeries(new_series, self.freq_str())
 
+    def diff(self, n: Optional[int] = 1,
```

Comment:
You can put each argument on a different line (including `self`).

Reply:
Done